vendor/engines/split_accounts/app/views/facility_accounts/account_fields/split_accounts/_splits.html.haml

```diff
@@ -0,0 +1,17 @@
+- available_subaccounts = Account.excluding_split_accounts.filter_global_account_types
+= javascript_include_tag "split_account_splits"
+
+%h4 Subaccounts
+
+%div{data: {subaccounts: true}}
+  = f.nested_fields_for :splits, class_name: "SplitAccounts::Split" do |ff|
+    .well
+      = ff.input :subaccount_id, collection: available_subaccounts, wrapper_html: {class: "input-inline"}
+      = ff.input :percent, input_html: {min: 0, max: 100, data: {percent: true}}, wrapper_html: {class: "input-inline"}
+      = ff.input :extra_penny, wrapper_html: {class: "input-inline"}
```

Comment:
Couple thoughts:

* Would it make sense to default the first subaccount's extra penny field to true?
* Should we have one (or maybe even two) subaccounts exposed when you first land on the page?
* Can the extra penny be a radio button? Or is that too complicated?